test/pspm_find_sounds_test.m

```diff
@@ -16,29 +16,32 @@
     channel_action = {'none', 'add', 'replace'};
   end
   methods(Test)
-    function invalid_input(this)
-      % file does not exist
-      this.verifyWarning(@() pspm_find_sounds(''), 'ID:file_not_found');
+    function invalid_input(this) % file does not exist
+      this.verifyWarning(@() pspm_find_sounds(''), 'ID:nonexistent_file');
       % create empty file
       fn = pspm_find_free_fn(this.testdata_fn, '.mat');
       fclose(fopen(fn, 'w'));
       % test with invalid pspm file
-      this.verifyWarning(@() pspm_find_sounds(fn), 'ID:invalid_input');
+      this.verifyWarning(@() pspm_find_sounds(fn), 'ID:invalid_file_type');
       % test with data without a snd channel
       c{1}.chantype = 'scr';
       pspm_testdata_gen(c, 10, fn);
-      this.verifyWarning(@() pspm_find_sounds(fn), 'ID:no_sound_chan');
+      this.verifyWarning(@() pspm_find_sounds(fn), 'ID:non_existing_chantype');
```

Comment:
I also saw `unexpected_chantype` in other functions... are they different or not..?

Reply:
The warning ID for a non-existing chantype is generic now, not specific for each channel type.